cmake/Config.cmake.in

```diff
@@ -0,0 +1,4 @@
+@PACKAGE_INIT@
+
```

Comment:
Since this package depends on other packages (OpenSSL, Threads) we should call `find_package` with corresponding conditions here, see example:
* https://github.com/hunter-packages/ogles_gpgpu/blob/07b3af57c64feb356a9e3c0f3fb528e4d9e72043/cmake/Config.cmake.in#L3-L28

Reply:
Yes. I finally found that is the reason my build continues to fail in hunter. Should have caught that sooner.